### ISSUE TITLE
inloadout not working

### ISSUE THREAD

**Issue by duckmanBR** [DestinyItemManager/DIM#3104]
is:inloadout is only working if I refresh the app with the loadout popup open, and then it only highlight items until the next refresh.

**Comment by bhollis** [comment]
Yeah, the way I re-implemented this really doesn't work with the new React inventory. I can fix it but it requires moving loadouts into Redux as well. Might have to remove the feature again in the meantime.
